### PR TITLE
Replace deprecated components [#10341]

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Replacing picto cards and feature cards on feature pages and heroes on comparison pages.

Most of the comparison pages weren't using the hero macro and already had a semi-customized hero, so I just made it a fully custom element only used on these pages. I think we may be creating a new simple hero component soon and could update these pages again, but for now I think just de-coupling them from the old hero is a good step.

This also replaces most of the feature icons with the ones we added to Protocol a long time ago, finally making good on a promise.

## Issue / Bugzilla link
#10341 

## Testing
http://localhost:8000/firefox/browsers/compare/ et al
http://localhost:8000/firefox/features/ et al

### CODE REVIEW

File: bedrock/firefox/templates/firefox/browsers/compare/index.html [hunk 1]
```diff
@@ -19,26 +21,21 @@
...
-        <p>{{ ftl('compare-index-looking-for-a-better-browser', fallback='compare-index-looking-for-a-better-fallback') }}</p>
-      </div>
-
-      {{ download_firefox(dom_id='compare-hero-download', download_location='primary cta') }}
-
+{% call split(
+    image_url='img/firefox/compare/compare-browser-windows.png',
+    include_highres_image=True,
+    block_class='',
+    theme_class='',
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/browsers/compare/index.html:28]: Do these need to be defined as empty strings? (the defaults seem to be `None` in the macro)

File: bedrock/firefox/templates/firefox/features/includes/features-content.html [hunk 2]
```diff
@@ -1,82 +1,121 @@
...
-      </div>
-    </section>
+  <ul class="mzp-l-columns mzp-t-columns-three">
+    {% call picto(
+      base_el='li',
+      image_url='protocol/img/icons/brand/orange/speedometer.svg',
+      image_width='32',
+      body=True,
+    ) %}
+      <p><a href="{{ url('firefox.features.fast') }}" rel="external">{{ ftl('features-shared-browse-faster') }}</a></p>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/features/includes/features-content.html:13]: Remove `rel="external"` on links here that are same site?

**Comment by alexgibson** [comment]
The hero images on the following three pages look a bit too large to me?

- http://localhost:8000/en-US/firefox/features/block-fingerprinting/
- http://localhost:8000/en-US/firefox/features/translate/
- http://localhost:8000/en-US/firefox/privacy/safe-passwords/

image

**Review by alexgibson — CHANGES_REQUESTED** [review]
There's a lot of changes here, but I pulled each page locally and everything looks respectable (bar the large hero images on a few pages). I took a pass through the code changes and only noticed a couple of nits.

Not a blocker for this PR, but it would be great if the new split macro supported optional lazy-loading like the feature-card macro does. I don't think it effects any images on these pages in question though. r+wc 🖼️

File: bedrock/firefox/templates/firefox/browsers/compare/index.html (hunk 1 above)
Comment (craigcook) [reply bedrock/firefox/templates/firefox/browsers/compare/index.html:28]: Oops, that was a leftover from pasting that I neglected to remove.

**Comment by craigcook** [comment]
The huge images are because I hadn't included the `mzp-l-split-media-constrain-height` class, and never noticed because of my ancient low-res monitor. Fixed now!

**Review by alexgibson — APPROVED** [review]
Looking good on my monitor! r+

**Pull request merged**